en/upgrades/decimal-upgrade-to-2-03/2-03-changes.rst

```diff
@@ -0,0 +1,183 @@
+2.03 Changelog
+^^^^^^^^^^^^^^
+
+Changes / additions to IATI Standard for version 2.03
+
+Note: Version 2.03 was released on Monday 19th February 2018.
+
+.. contents::
+
+
+.. _2_03_schema_changes:
+
+Schema Changes
+==============
+
+A full diff of all schema changes in 2.03 can be found in the `IATI GitHub <https://github.com/IATI>`__
+
+- `Activities Schema <https://github.com/andylolz/IATI-Schemas/pull/1/commits/97732bdaa5543aa657d2a667f5b23fdfd9ac8f36>`__
+
+- `Organisations Schema <https://github.com/andylolz/IATI-Schemas/pull/1/commits/fa9898cd7f867741aa8a06d90c9b63d30e2d5937>`__
+
+- `Activities and Organisations (Common) Schema <https://github.com/andylolz/IATI-Schemas/pull/1/commits/0b51e6ee4d2231c64857216e5cb675ac8ccbf305>`__
+
+- `Embedded Codelists <https://github.com/IATI/IATI-Codelists/tree/version-2.03dev>`__
+
+- `Non-Embedded Codelists <https://github.com/IATI/IATI-Codelists-Nonembedded/tree/version-2.03dev>`__
+
+- `Rulesets <https://github.com/IATI/IATI-Rulesets/tree/version-2.03dev>`__
+
+Activities Schema
+-----------------
+
+- Added new @crs-channel-code attribute for the participating-org element (`#338 <https://github.com/IATI/IATI-Schemas/issues/338>`__) (`discussion link <https://discuss.iatistandard.org/t/crs-channels-of-delivery-included-2-03/857>`__)
+ 
+- Added new tag element (as a child of the iati-activity element) (`#342 <https://github.com/IATI/IATI-Schemas/issues/324>`__) (`discussion link <https://discuss.iatistandard.org/t/non-statistical-secondary-sectors-excluded-2-03/849>`__)
+ 
+- Updated definition: recipient-[country|region] / sector @percentage attribute (`#343 <https://github.com/IATI/IATI-Schemas/issues/343>`__) (`discussion link <https://discuss.iatistandard.org/t/boundary-values-for-percentages-included-2-03/843>`__)
+ 
+- Updated definition: sector @vocabulary attribute (`#344 <https://github.com/IATI/IATI-Schemas/issues/344>`__). The % sign was removed.
+ 
+- Updated definition: [transaction aid-type/default-aid-type] @code attribute (`#345 <https://github.com/IATI/IATI-Schemas/issues/345>`__) (`discussion link <https://discuss.iatistandard.org/t/add-vocabularies-to-aid-type-included-2-03/847>`__)
```

Comment:
Use of `/` makes this look like a single XPath when it's not - being comma-separated would make this clearer (here and elsewhere)